lib/steps.js

```diff
@@ -64,6 +64,7 @@ Before(async (scenario) => {
   scope.toDownload = '';
   scope.device = 'pc';
   scope.activate = click_with[ scope.device ];
+  scope.target_question_id = 'null';
```

Comment:
Where do we use this variable? 

Also, it seems weird to use 'null' as the default string. I think it would be better practice to use `null` as a keyword or to use an empty string.

Reply:
1. Good catch.
1. That was definitely not supposed to be a string
1. It should not be needed anymore. Something just got lost in translation.